src/Storage/Migration/v0.06/01-functions-and-procedures.sql

```diff
@@ -0,0 +1,135 @@
+CREATE OR REPLACE FUNCTION storage.updateinstance_v2(
+        _alternateid UUID,
+        _toplevelsimpleprops JSONB,
+        _datavalues JSONB,
+        _completeconfirmations JSONB,
+        _presentationtexts JSONB,
+        _status JSONB,
+        _substatus JSONB,
+        _process JSONB,
+        _lastchanged TIMESTAMPTZ,
+        _taskid TEXT)
+    RETURNS TABLE (updatedInstance JSONB)
+    LANGUAGE 'plpgsql'	
+AS $BODY$
+BEGIN
+    IF _datavalues IS NOT NULL THEN
```

Comment:
One observation. It's not possible to update datavalues and process (examples) in one call. There probably aren't any need for that exact example, but have we considered all permutations?

Reply:
I'm slightly worried it's possible to use this incorrectly.

Reply:
I have tried to cover all current combinations. Process is supported both stand alone and combined with status. (And lastchanged* is added to all combinations.)